From 0c28654bad4331d0558fec049c9bc8e80210a0c3 Mon Sep 17 00:00:00 2001
From: Scott Hollwedel <scott@thistle.tech>
Date: Tue, 23 Aug 2022 10:31:23 -0700
Subject: [PATCH] Generate a dynamic library

---
 Cargo.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Cargo.toml b/Cargo.toml
index 3552e93..f71e751 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -35,4 +35,4 @@ cbindgen = "0.19.0"
 
 [lib]
 name = "rustls_ffi"
-crate-type = ["lib", "staticlib"]
+crate-type = ["lib", "cdylib", "staticlib"]
